### PR TITLE
Installation Guide README.md

### PR DESCRIPTION
**Pull request by MariusLang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,8 +31,24 @@ The code is originally a port from the C# [nMQTT](https://www.openhub.net/p/nMQT
 
 Please read the changelog for details related to specific versions.
 
-
-
-
+## Installation
```
Comment (shamblett) [inline README.md:34]: This is OK< but 'running' here only applies to flutter platforms and specifically IOS and Android devices, the client can be used in many other ways, I think we should say something like 'If you are using the client in a flutter environment on Android or IOS devices then the following device permission settings may be necessary'

**Comment by shamblett** [comment]
I'm happy to add this to the README, didn't think of adding an installation section, I've added a minor comment.

**Comment by MariusLang** [comment]
Sure, you're right. I added your suggestion to the README.

**Comment by shamblett** [comment]
OK thanks, this will be in the next release of the package.

**Pull request merged**